### PR TITLE
Added OS Upgrade information.

### PR DESCRIPTION
**Pull request by mikemorrell2**
I added new information to the OS Upgrades section of the Silverblue Guide document.

### CODE REVIEW

**Comment by mikemorrell2** [comment]
I have reviewed and made a few improvements myself. I would welcome an outside review for content accuracy, spelling, and grammar. Thanks.

File: modules/ROOT/pages/silverblue-guide.adoc [hunk 1]
```diff
@@ -179,7 +179,64 @@ NOTE: The prompts as displayed, are normal for my system setup. They will very l
...
+
 === Rolling Back the OS
+If something went wrong or is not working correctly with a new  upgrade, then it
+may be necessary to rollback the upgrade until a fix is available. To rollback
+to the previous OS deployment, run the command `sudo rpm-ostree -r rollback`.
+
+You can also deploy previous OS images that you have pinned. Use the
+`rpm-ostree status` command to see the list of previous pinned deployments that
+are available. Then use the `sudo rpm-ostree deploy [version|branch|commit-id]`
+command to deploy the pinned image from the list.
```
Comment (miabbott) [inline modules/ROOT/pages/silverblue-guide.adoc]: This section is not entirely accurate.

You are able to deploy previous verseions that exist on the remote repo, not just those that have been pinned.

You can get a list of available versions by doing `ostree pull --commit-metadata-only --depth=-1 fedora:fedora/29/x86_64/silverblue` followed by `ostree log fedora:fedora/29/x86_64/silverblue`.  You can deploy any version or commit ID found in that log.

Pinning a deployment is a different operation; it is supposed to be used when you want to keep a particular version around after the host has been upgraded.  You can pin a deployment with `ostree admin pin`.

Please update this section with these details.  Thanks!

**Comment by mikemorrell2** [comment]
Okay. I have make correction to the section about deploying previous pinned deployment.

**Comment by miabbott** [comment]
Thanks for the contribution!

**Pull request merged**